src/api/status/src/views/status.hbs

```diff
@@ -35,8 +35,6 @@
     <div class="row">
```

Comment:
Might want to add some `row-gap` here, otherwise, the card rows are colliding with each other 
![image](https://user-images.githubusercontent.com/58233223/163914043-d898c8c8-26ae-4245-9963-7e30183474c0.png)

Reply:
Ignore that, what a timing lol. I posted a few seconds after your commit.

Reply:
haha